client-cli/src/test/java/org/eclipse/dataspaceconnector/registration/cli/RegistrationUrlResolverTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.registration.cli;
+
+import org.eclipse.dataspaceconnector.iam.did.spi.document.DidDocument;
+import org.eclipse.dataspaceconnector.iam.did.spi.document.Service;
+import org.eclipse.dataspaceconnector.iam.did.spi.resolution.DidResolver;
+import org.eclipse.dataspaceconnector.iam.did.web.resolution.WebDidResolver;
+import org.eclipse.dataspaceconnector.spi.result.Result;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class RegistrationUrlResolverTest {
+
+    public static final String SERVICE_TYPE = "RegistrationUrl";
+
+    DidResolver didResolver = mock(WebDidResolver.class);
+    RegistrationUrlResolver urlResolver = new RegistrationUrlResolver(didResolver);
+
+    String did = "did:web:didserver";
```

Comment:
nit: maybe faker can be used same as another test. Not necessary here but just saw it as it was used in another test in this PR. So just a suggestion.

Reply:
sure, makes sense to use faker here